hildon/hildon-gtk.c

```diff
@@ -31,6 +31,24 @@
 #include "hildon-edit-toolbar-private.h"
 #include "hildon-private.h"
 
+/* Add placeholder implementations for Maemo-specific Gtk functions */
+#ifndef MAEMO_GTK
+void
+hildon_gtk_entry_set_placeholder_text           (GtkEntry *entry,
+                                                 const gchar *text)
```

Comment:
Arguments are not aligned, please fix.
Follow the style of other functions from this file.

Reply:
Fixed